src/System.Threading.Tasks.Dataflow/tests/Dataflow/EtwTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Threading.Tasks.Dataflow.Tests
     public class EtwTests
     {
         [Fact]
+        [ActiveIssue(3520, PlatformID.OSX)]
```

Comment:
Matt, the EtwTests is now failing on Linux as well, e.g. http://dotnet-ci.cloudapp.net/job/dotnet_corefx_linux_debug_tst/1052/testReport/junit/System.Threading.Tasks.Dataflow.Tests/EtwTests/TestEtw/.  Were you able to track down the script problem?